docs/01-Introduction.md

```diff
@@ -3,3 +3,10 @@
 ## 1.1 What is LORIS?
 
 ## 1.2 How does LORIS work?
+
+
+### Deidentification of imaging datasets
+
+All DICOM datasets transferred to the DCC must be free of any identifying
+  information (*e.g.* patient name). A tool can be provided to the sites to
+  facilitate deidentification. Please contact the LORIS team for details.
```

Comment:
or refer them to FAQ where we tell them to either use DiCAT or use the dcmodify command.